.travis.yml

```diff
@@ -5,8 +5,7 @@ node_js:
   - '8'
   - '6'
 install:
-  - 'if [ -n "${JEST}" ]; then yarn add --dev "jest@${JEST}" "babel-jest@${JEST}"; fi'
-  - 'if [ -n "${ESLINT}" ]; then yarn add "eslint@${ESLINT}"; fi'
+  - 'yarn add "eslint@${ESLINT}" "jest@${JEST}" "babel-jest@${JEST}"'
```

Comment:
if this isn’t done conditionally, won’t it trigger on a future, say, linting build or similar? I suppose we could add the condition then too tho.

Reply:
Most projects I've seen have an explicit `include` that runs eslint, since you don't need to run that more than once. My projects also have one for coverage, and one for `npx aud` for an audit check. This project doesn't seem to have that yet, however.